assembl/static2/js/app/graphql/mutations/updateResource.graphql

```diff
@@ -1,10 +1,14 @@
+#import "../fragments/Resource.graphql"
+
 mutation updateResource(
   $doc: String
   $id: ID!
   $image: String
+  $lang: String
   $titleEntries: [LangStringEntryInput]!
   $textEntries: [LangStringEntryInput]
   $embedCode: String
+  $order: Float
 ) {
   updateResource(
     doc: $doc
```

Comment:
Don't you need to add lang variable here?

Reply:
no, it is used only for mutation result (`Resource` fragment)

Reply:
So is this really required in the createResource mutation?